docs/source/release-notes/2021-1-3-Eos.md

```diff
@@ -0,0 +1,267 @@
+# 2021.1.3 Eos Release Notes
+
+This version of Vircadia is an update to 2021.1.2. It is compatible with the 2020.1.0 Eos protocol. This version was released on September 22nd, 2021.
+
+## [Native Interface & Domain Server](https://github.com/vircadia/vircadia/) (Codename Athena)
+
+### Enhancements
+
+- [**linux**] Add crash handler support for Linux. [#365](https://github.com/vircadia/vircadia/pull/365)
+- [**windows**] Update WebRTC package for Windows. [#1258](https://github.com/vircadia/vircadia/pull/1258)
+-  Rename "HifiSockAddr" to "SockAddr". [#1283](https://github.com/vircadia/vircadia/pull/1283)
+-  Improve Qt handling in CMake. [#936](https://github.com/vircadia/vircadia/pull/936)
+- [**docs**][**housekeeping**] Add "gren" configuration for easy changelog generation. [#1226](https://github.com/vircadia/vircadia/pull/1226)
+- [**docs**] Add minimal documentation for classes declared in script-engine. [#1223](https://github.com/vircadia/vircadia/pull/1223)
+- [**docs**][**renderer**] Material System Improvements Part 2: Reference textures/materials by entity ID. [#1102](https://github.com/vircadia/vircadia/pull/1102)
+-  Put generated resources and docs into the build directory. [#943](https://github.com/vircadia/vircadia/pull/943)
+-  Don't use LeapMotion by default. [#1268](https://github.com/vircadia/vircadia/pull/1268)
+-  Replace uses of endl on Qt classes with Qt::endl [#1270](https://github.com/vircadia/vircadia/pull/1270)
+- [**scripting api change**] Drop remaining references to "QScriptEngineDebugger". [#1206](https://github.com/vircadia/vircadia/pull/1206)
+-  Archive CMake logs on Github Actions for debugging. [#1274](https://github.com/vircadia/vircadia/pull/1274)
+-  Disable cloning of Git submodules with GitHub Actions. [#1276](https://github.com/vircadia/vircadia/pull/1276)
+-  Create app - Add "Reset Rotation" button. [#1316](https://github.com/vircadia/vircadia/pull/1316)
+-  Update "Explore" app to load Metaverse places in parallel. [#1315](https://github.com/vircadia/vircadia/pull/1315)
+-  Make the HUD bubble/shield button optional. [#1314](https://github.com/vircadia/vircadia/pull/1314)
+-  Log the location of the primary resources file. [#1309](https://github.com/vircadia/vircadia/pull/1309)
+-  Add "Talk" link to "Help" menu. [#1288](https://github.com/vircadia/vircadia/pull/1288)
+-  Update OpenVR to 1.16.8. [#1322](https://github.com/vircadia/vircadia/pull/1322)
+-  Reduce stale-bot's aggressiveness. [#1285](https://github.com/vircadia/vircadia/pull/1285)
+-  Make Zone Selector for "renderWithZones" in "Create" app HMD Compliant. [#1252](https://github.com/vircadia/vircadia/pull/1252)
+- [**web sdk**] Add WebRTC Signaling Channel. [#1219](https://github.com/vircadia/vircadia/pull/1219)
```

Comment:
```suggestion
```
This is in the `webapp` branch from a while back. ... Move to "In Development" section?